silverorange/ambiguous-class-name-detector
# Check namespaced classes

Fixes #1 


Testing:

1. check out this PR on berna in `/so/packages/ambiguous-class-name-detector/work-[me]`
2. `composer install`
3. go to emrap project on berna, make sure branch is up to date with master
4. `composer install`
5. run `/so/packages/ambiguous-class-name-detector/work-[me]/bin/check-for-ambiguous-class-names`
6. It should show the following message:

> One or more ambiguous class names detected:
>
>  "silverorange\Emrap\MigrateToWeekly\MigrateToWeekly" in "./system/fix-snacks-artwork/FixSnackArtworks.php" and "./system/migrate-to-weekly/MigrateToWeekly.php"
